src/parser.rs

```diff
@@ -173,6 +166,7 @@ fn check_get_cell() {
     }
 }
 
+
```

Comment:
Extra empty line. However I'm thinking about moving test in their own "tests" folder. It'll be better this way.